src/pages/research.astro

```diff
@@ -0,0 +1,1131 @@
+    preprintOrArchiveUrl:
+      "https://nixos.org/~eelco/pubs/immdsd-icse2004-final.pdf",
+    doiOrPublisherUrl: "10.1109/ICSE.2004.1317480",
+    publicationInfo: {
+      type: "conference",
+      conference:
+        "26th International Conference on Software Engineering (ICSE 2004)",
+      location: "Edinburgh, Scotland",
+      publisher: "IEEE Computer Society",
+      pages: "583–592",
+    },
+  },
+  {
+    title: "Integrating Software Construction and Software Deployment",
+    authors: [{ name: "Eelco Dolstra" }],
+    year: 2003,
+    abstract:
+      "Classically, software deployment is a process consisting of building the software, packaging it for distribution, and installing it at the target site. This approach has two problems. First, a package must be annotated with dependency information and other meta-data. This to some extent overlaps with component dependencies used in the build process. Second, the same source system can often be built into an often very large number of variants. The distributor must decide which element(s) of the variant space will be packaged, reducing the flexibility for the receiver of the package. In this paper we show how building and deployment can be integrated into a single formalism. We describe a build manager called Maak that can handle deployment through a sufficiently general module system. Through the sharing of generated files, a source distribution transparently turns into a binary distribution, removing the dichotomy between these two modes of deployment. In addition, the creation and deployment of variants becomes easy through the use of a simple functional language as the build formalism.",
+    publicationInfo: {
+      type: "conference",
+      conference:
+        "11th International Workshop on Software Configuration Management (SCM-11)",
+      location: "Portland, Oregon, USA",
+      pages: "102–117",
+    },
+    preprintOrArchiveUrl: "https://nixos.org/~eelco/pubs/iscsd-scm11-final.pdf",
+  },
+];
```

Comment:
Maybe this could be split up into a separate content collection, see https://docs.astro.build/en/guides/content-collections/

Reply:
I do not want to spend a lot of time on putting every entry in its own file right now, because that interferes with my own personal workflow for adding and modifying entries and I would also have to add an index to keep them in the intended order.

Astro on the other hand does not seem to like collections where all the content is in one file though (that's what I would like to do at least for now), so that also does not work.

Basically I'd would prefer to keep this as is now, or just move all entries into another file, and then see later on if making it a collection with each entry in its own file makes sense.